### ISSUE TITLE
Add contributors to people who are awesome file!

### ISSUE THREAD

**Issue by amiefairs** [dasaderi/WhyNotOpenScience#14]
Have you contributed? If so you are going in our thank you file! We want to make sure we remember and acknowledge everyone who has helped with the project. Your name will be listed below, but if you don't see it, add it yourself and you'll be thanked forever in Github's memory :)

**Comment by dasaderi** [comment]
@amiefairs I think we could add a Thank_you_file.md to the repo. Is that what you meant? :)

**Comment by amiefairs** [comment]
@dasaderi yes exactly! I have created one already in a 'thank you' folder in the repo (in thank_you -> awesomePeople.md), I'm going to add everyone who has helped to it in the next few days :)
